### ISSUE TITLE
Sublime 3 (fresh and old instalation) expand_fqcn and goto_definition_scope don't work

### ISSUE THREAD

**Issue by leandromatos** [erichard/SublimePHPCompanion#80]
I've been trying to use expand_fqcn and goto_definition_scope for a few days. I've tried to disable all plugins and it still doesn't work. I've also tried to use a new installation without plugins without success. Anyone else has had this problem?

**Comment by phppirate** [comment]
Yes, I have this problem as well.

I had switched back to Sublime after being a long time PHP Storm user, but because this feature has stopped working I am back to Storm.

Really hoping this gets fixed soon.

**Comment by iamanders** [comment]
Duplicate of #76 ?

**Comment by leandromatos** [comment]
Yes, this issue was opened after #76 closed.

As the problem there was not solved, I opened this issue again, but, a few days they opened #76.

**Comment by erichard** [comment]
I close this one as duplicate
